### ISSUE TITLE
I see how it is...

### ISSUE THREAD

**Issue by agarian-2** [Faris90/arrasserv#1]
So, it's you. You that the community has been talking about. You that infiltrated my Cloud9 workspace. I've been looking for you. From what I see here, this server belongs to a friend of mine. In fact, I see you get your kicks out of stealing various people's servers, as I recognize some of the tanks. Fair point, but it's time you payed the price. You've joined my server a few times, meaning I have your IP address. With that, I can find your location, find out who you are; and god knows what somebody could do with that info. I suggest you stop stealing other people's servers. Hell, you're lucky I haven't reported you for doing such a thing, that alone could get you in some deep trouble. Get the message? If I ever see you steal another server, I WILL report you (and take note, I still do have your IP, so I could hand that over to the Github developers as well). Let this be a warning; from me and the arras.io community.

**Comment by AC5230** [comment]
BE WARNED: You'll be dying in a fire. Nobody likes you, and you should be put up for adoption. NOBODY, and I mean _NOBODY_, goes and copies Hellcat's server. Or anyone else's servers. And takes the copies and says they're "original games" What are you, gay!?!

But no, you need to think before you do things. You drove us all insane, and now you're paying the price.
Say goodbye to Github as I may or may not report you. (Here's the thing: I probably will.)

**STOP AND THINK** BEFORE YOU DO THINGS

**Comment by Koleotank** [comment]
And also faris wheel, i banned you from the arrasio subreddit, you keep being such a nuisance to me and the community, like a week ago or so because of your BS a huge, i mean, REALLY FUCKING HUGE public outcry happened. I had to ban you as you also were fucking around with all of us.

If i ever catch you stealing my server on glitch... be warned.

**Comment by CancelX1** [comment]
i agree about this, taking peoples servers and then taking them for your own sick uses is not right
i wish you be banned, and then posting them on reddit hey all look at this completely origanal server i made. so yeah

**Comment by geoffrey1900** [comment]
hes so retarded i cant see why this person doesnt have more projects to piss more users off...

**Comment by Koleotank** [comment]
he almost fucking killed my devbot with his stealing kicks
